docker/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM dealii/dealii:v8.5.pre.4-gcc-mpi-fulldepsmanual-debugrelease
+
+LABEL maintainer <rene.gassmoeller@mailbox.org>
+
+# Build aspect
+RUN git clone https://github.com/geodynamics/aspect.git ./aspect && \ 
+    mkdir aspect/build && \
+    cd aspect/build && \
+    cmake -DCMAKE_BUILD_TYPE=Release \
+          -DDEAL_II_DIR=$HOME/deal.II-install \
+          .. && \
+    make -j4 >log 2>&1 && \
+    mv aspect $HOME/aspect/ && \
```

Comment:
Hi Rene, I just built the image on my local laptop and noticed that the build info is being redirected to a log file

`make -j4 >log 2>&1`

which is then discarded.

This is only a suggestion but I think that it is informative to know the status of the build, whether it is 10% done or 90% done and so on. However, I may be alone on this.